examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6558,15 +6558,15 @@ endpoint 1 {
     ram      attribute currentScene default = 0x00;
     ram      attribute currentGroup default = 0x0000;
     ram      attribute sceneValid default = 0x00;
-    ram      attribute nameSupport;
+    ram      attribute nameSupport default = 0x80;
```

Comment:
Why is this stored in RAM at all, if we actually always force a contant value in cluster init?

Reply:
As noted in the Todo, this is currently forced on but shouldn't, a subsequent PR will make it read from Zap config instead of forced in the init.